models/generated/directory/directory.go

```diff
@@ -295,3 +296,17 @@ type Gateway struct {
 	DnsNameserver  []string       `bson:"dns_nameserver" json:"dns_nameserver"`
 	FreeToUse      bool           `bson:"free_to_use" json:"free_to_use"`
 }
+
+type FarmerIP struct {
+	net.IP        `bson:"ip" json:"ip"`
+	ReservationID schema.ID `bson:"reservation_id" json:"reservation_id"`
+}
+
+func NewFarmerIP(ip net.IP) (FarmerIP, error) {
```

Comment:
What is that? and what it's used for?

I think this kind of constructors where used with the schema generated types, we don't need it anymore.